silkrpc/commands/debug_api.cpp

```diff
@@ -128,10 +128,10 @@ boost::asio::awaitable<void> DebugRpcApi::handle_debug_get_modified_accounts_by_
 
     try {
         ethdb::TransactionDatabase tx_database{*tx};
-        const auto start_block_number = co_await core::get_block_number(start_block_id, tx_database);
-        const auto end_block_number = co_await core::get_block_number(end_block_id, tx_database);
+        const auto start_block_number = co_await core::get_block_number(start_block_id, tx_database, false);
```

Comment:
In order to increase readability and clarity, I suggest to use a comment with the name for the `latest_required` field at each `core::get_block_number` call site:
```
... = co_await core::get_block_number(start_block_id, tx_database, /*latest_required=*/false);
```